<robot name="robot">
  <link name="base_footprint">
    <visual>
      <geometry>
        <box size="15 3 2"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 -1"/>
      <material name="white">
        <color rgba="1 1 1 1"/>
      </material> 
    </visual>
  </link>

  <link name="wheel_1">
    <visual>
      <geometry>
        <cylinder length="0.3" radius="0.875"/>
      </geometry>
      <origin rpy="1.57 -1.57 0" xyz="4.4 1.5 -2"/>
      <material name="black">
        <color rgba="1 1 1 1"/>
      </material> 
    </visual>
  </link>

  <link name="wheel_2">
    <visual>
      <geometry>
        <cylinder length="0.3" radius="0.875"/>
      </geometry>
      <origin rpy="1.57 -1.57 0" xyz="4.4 -1.5 -2"/>
      <material name="black">
        <color rgba="1 1 1 1"/>
      </material> 
    </visual>
  </link>

  <link name="wheel_3">
    <visual>
      <geometry>
        <cylinder length="0.3" radius="0.875"/>
      </geometry>
      <origin rpy="1.57 -1.57 0" xyz="-4.4 1.5 -2"/>
      <material name="black">
        <color rgba="1 1 1 1"/>
      </material> 
    </visual>
  </link>

 <link name="wheel_4">
    <visual>
      <geometry>
        <cylinder length="0.3" radius="0.875"/>
      </geometry>
      <origin rpy="1.57 -1.57 0" xyz="-4.4 -1.5 -2"/>
      <material name="black">
        <color rgba="1 1 1 1"/>
      </material> 
    </visual>
  </link>

  <joint name="base_to_wheel1" type="fixed">
    <parent link="base_footprint"/>
    <child link="wheel_1"/>
    <origin xyz="0 0 0"/>
  </joint>

  <joint name="base_to_wheel2" type="fixed">
    <parent link="base_footprint"/>
    <child link="wheel_2"/>
    <origin xyz="0 0 0"/>
  </joint>

  <joint name="base_to_wheel3" type="fixed">
    <parent link="base_footprint"/>
    <child link="wheel_3"/>
    <origin xyz="0 0 0"/>
  </joint>

  <joint name="base_to_wheel4" type="fixed">
    <parent link="base_footprint"/>
    <child link="wheel_4"/>
    <origin xyz="0 0 0"/>
  </joint>	

</robot>
